lib/mayaUsd/utils/util.h

```diff
@@ -690,6 +690,8 @@ void AddMayaExtents(
 MAYAUSD_CORE_PUBLIC
 SdrShaderNodePtrVec GetSurfaceShaderNodeDefs();
 
+bool isShape(const MDagPath& dagPath);
```

Comment:
I think we shoudl export it by adding MAYAUSD_CORE_PUBLIC, like the other functions.